Library/Formula/rofs-filtered.rb

```diff
@@ -0,0 +1,28 @@
+class RofsFiltered < Formula
+  homepage "https://github.com/gburca/rofs-filtered/"
+  url "https://github.com/gburca/rofs-filtered/archive/rel-1.5.tar.gz"
+  sha256 "4c97a85e7993945e417c45cc80fcb8ad780f310142b8f94e8b3bed005733a698"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "intltool" => :build
+  depends_on "gettext" => :build
```

Comment:
Run an ` otool -L ` on the binary. If any links are shown to any of the deps, they aren't purely build-time deps.